docs/Preferences.md

```diff
@@ -161,3 +161,15 @@ The different options are:
 Here is a demonstration of the effects of the setting on hovering. The above schema is used in the example:
 
 ![Changing the documentation type setting changes which text the hover shows when hovering over an element that is in a schema document](./images/Preferences/HoverDocumentationQuickDemo.gif)
+
+## Extension JARs
+
+[LemMinX](https://github.com/eclipse/lemminx), the underlying language server that vscode-xml uses to provide most of its functionality,
+is extensible through Java's SPI mechanism.
+Please see [the extension development documentation](https://github.com/eclipse/lemminx/blob/master/docs/LemMinX-Extensions.md) for more information on how this works.
+
+JARs can be contributed to the LemMinX classpath using the `xml.extension.jars` preference.
+These paths can include globs.
+This feature is only intended to be used for LemMinX extension development purposes.
+Distributing vscode-xml extensions is best done through the mechanism described in the
+[vscode-xml extension development documentation](https://github.com/redhat-developer/vscode-xml/wiki/Extensions).
```

Comment:
It should be nice if we could explain how to use this settings with lemminx-maven. I don't know if the lemminx-maven zip is available https://github.com/eclipse/lemminx-maven/pull/148#issuecomment-704780003

If it is available it should be nice to add a section here and provide a screenshot with completion maven.

Reply:
I don't know how to get lemminx-maven to work. Is there a way to generate an uber jar, or do I need to include all the separate jars with a glob?

Reply:
I tried loading all the jars in the generated zip from a project build, but it doesn't work. I get an exception about invalid certificates. Liquidbase works fine, though.

Reply:
Could you report this issue in lemminx-maven please.

Reply:
I would prefer explain how to configure lemminx-maven which is an Eclipse Project, but if it doesn't work for the moment, please create an issue for that (add lemminx-maven sample in the doc).